gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -77,9 +77,27 @@ to generate set of instructions that includes a command block with the registrat
 // tag::reg-command-intro[]
 To register a Flex Gateway with Anypoint Platform, you must enter the registration command and then the start command. Each command includes information specific to your Anypoint Platform instance and *must be updated before* executing. See <<Before You Begin>> for more information on how to find the information you will need.
 //end::reg-command-intro[]
+//
+// tag::k8s-connected-intro[]
+In this mode, the gateway is managed entirely through Anypoint Platform, where you can: 
+
+* Deploy APIs to your cluster.
+* Apply policies and other configurations.
+* Create all resources through Anypoint Platform. 
+
+Resource creation through `kubectl apply` is restricted to specific resource types.
```

Comment:
In targets that are not k8s (like docker and vms) `kubectl apply` is not used

Reply:
should we mention ☝️ in the docs?

Reply:
I think we should try to say something that includes all targets. Maybe something like `Resource creation is restricted to specific resource types.` omitting how they would be created, because that is different in each target

Reply:
I think that later in the docs we specify how this works in each target, so we might not want to go into those details here